meson.build

```diff
@@ -66,6 +66,12 @@ if use_docs
         prog_rst2man = find_program('rst2man', 'rst2man.py')
 endif
 
+#
+# Config: install-tests
+#
+install_tests = get_option('install-tests')
+test_install_dir = get_option('prefix') / 'share/dbus-broker/tests'
```

Comment:
No particular reason at all. As said, I just picked the one that was used the most in the packages I went through. I'll switch it over to `/usr/lib/`, since it indeed makes more sense in this case.